volumina/__main__.py

```diff
@@ -0,0 +1,80 @@
+import argparse
+import contextlib
+import signal
+import sys
+
+import numpy
+import xarray
+from PyQt5.QtWidgets import QApplication
+
+from volumina import __version__
+from volumina.api import Viewer
+from volumina.colortables import default16_new
+from volumina.pixelpipeline.datasources import ArraySinkSource, ArraySource
+from volumina.layer import ColortableLayer, GrayscaleLayer
+
+
+@contextlib.contextmanager
+def volumina_viewer():
+    app = QApplication(sys.argv)
+    v = Viewer()
+    v.editor.setInteractionMode("navigation")
+    yield v
+    try:
+        signal.signal(signal.SIGINT, lambda *_: app.quit())
```

Comment:
There is actually a bug when ctrl+c is pressed when no lazyflow is installed. Due to the incorrect override of protected `_work_queue` in `prioritizedThreadPool.py`. I don't think it's super relevant for this PR, but it would be nice to track this with an issue.

Reply:
geee, would you care to open this issue?

Reply:
https://github.com/ilastik/ilastik/issues/2399